get.sh

```diff
@@ -631,6 +632,7 @@ testJavaVersion()
 			TEST_JDK_HOME=${java_dir}/../
 			echo "TEST_JDK_HOME=${TEST_JDK_HOME}" > ${TESTDIR}/job.properties
 		else
+			echo "TEST_JDK_HOME=${TEST_JDK_HOME}" > ${TESTDIR}/job.properties
```

Comment:
We are using javac_path to find TEST_JDK_HOME. There is no point to set TEST_JDK_HOME if we cannot find javac. The build will fail.